packages/frontend/src/pages/campaigns/index.tsx

```diff
@@ -46,6 +47,18 @@ export const Campaigns = () => {
     const [currentPage, setCurrentPage] = useState(1);
     const { data, totalPages } = usePagination(results, currentPage, 4);
 
+    const handleOrderingChange = (orderingState: SelectOption) => {
```

Comment:
Should wrap this with `useCallback`.

Reply:
fixed: 0ee432617c7502aaf92c64b8bfe336bf551386a0